dts/msm8916/rules.mk

```
@@ -15,6 +15,7 @@ DTBS += \
 	$(LOCAL_DIR)/msm8916-huawei-hwt1a21l.dtb \
 	$(LOCAL_DIR)/msm8916-huawei-y635-l01.dtb \
 	$(LOCAL_DIR)/msm8916-lg.dtb \
+        $(LOCAL_DIR)/msm8916-vivo-y21l.dtb \
```

```suggestion
	$(LOCAL_DIR)/msm8916-vivo-y21l.dtb \
```

And sort it alphabetically.